src/electionguard/utils.py

```diff
@@ -14,11 +16,53 @@
 class ContestErrorType(Enum):
     """Various errors that can occur on ballots contest after voting."""
 
+    Default = "default"
     NullVote = "nullvote"
     UnderVote = "undervote"
     OverVote = "overvote"
 
 
+class ContestException(Exception):
```

Comment:
In an ideal world, these would not exist in this file. However, the amount of circular dependencies that arose in this PR were troublesome. This file allowed some sidestep.